src/objecttypes/templates/admin/core/objecttype/object_github_form.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'admin/base_site.html' %}
+
+{% block content %}
+    <h1>Import from URL</h1>
+    <div>
+        <form action="" method="POST" enctype="multipart/form-data">
+            {{ form.as_p }}
```

Comment:
I think the styling of the form fields can be adjusted to look like other django admin forms:
* labels should have the same width 
* margin between fields should be a little bigger

`admin/change_form/html` can be used as a reference